rllib/algorithms/ppo/tests/test_ppo.py

```diff
@@ -118,12 +119,12 @@ def test_ppo_compilation_and_schedule_mixins(self):
             .callbacks(MyCallbacks)
         )  # For checking lr-schedule correctness.
 
-        num_iterations = 2
+        num_iterations = 1  # TODO
 
         for fw in framework_iterator(config, with_eager_tracing=True):
-            for env in ["FrozenLake-v1", "MsPacmanNoFrameskip-v4"]:
+            for env in ["FrozenLake-v1"]:  # , "MsPacmanNoFrameskip-v4"]:
                 print("Env={}".format(env))
-                for lstm in [True, False]:
+                for lstm in [False]:  # , True]:
```

Comment:
clean up all the ``# TODOs`` and comments?

Reply:
done